app/models/organisation.rb

```diff
@@ -110,6 +114,18 @@ def social_media_links
     details["social_media_links"]
   end
 
+  def foi_exempt?
+    details["foi_exempt"]
+  end
+
+  def foi_contacts
+    links["ordered_foi_contacts"]
+  end
+
+  def ordered_contacts
```

Comment:
This means we can delete the `ordered_contacts` method at the bottom of this file.